code/game/objects/items/weapons/weaponry.dm

```diff
@@ -158,4 +158,40 @@ obj/item/weapon/wirerod/attackby(var/obj/item/I, mob/user as mob, params)
 
 
 
-
+/obj/item/weapon/switchblade
+	name = "switchblade"
+	icon_state = "switchblade"
+	desc = "A sharp, concealable, spring-loaded knife."
+	flags = CONDUCT
+	force = 20

```

Comment:
And I'm fine with that! 20 is what I would consider to be the minimally acceptable force for the weapon to have any value at all.

Keep in mind that in order to get one you have to spend *valuable* influence that isn't just to buy toys, but also to keep your gang going with spraycans.